cron/main.go

```diff
@@ -73,6 +79,11 @@ func main() {
 			panic(err)
 		}
 		sugar := logger.Sugar()
+		// deleting branch-protection
```

Comment:
maybe prefix this with FIXME:, we want to work in future with github to reconsider this decision.

Reply:
Agreed, Thanks